### PR TITLE
Update Catalog HTML and FO Transforms for NIST SP 800-53 Revision 5.2 Release

### PR DESCRIPTION
**Pull request by aj-stein-nist**
This work is part of usnistgov/oscal-content#134. I will not explicitly link it for now as to not close the issue. I will work with wendell to 1) merge this code here, and then 2) merge it upstream to main in [the upstream origin repo](https://github.com/usnistgov/oscal-xslt.git) accordingly.

### CODE REVIEW

**Comment by aj-stein-nist** [comment]
@wendellpiez this is ready for review and then we can merge into usnistgov/oscal-xslt later. Just to be transparent, my development/testing approach:

1. Use current [development version of the NIST SP 800-53 Revision 5 catalog (in `develop` not main)](https://raw.githubusercontent.com/usnistgov/oscal-content/a3231bb614510cd0832895a1e30fa7716add22a3/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml) to include implemented by and assurance props
2. Generate HTML from catalog with `rev5update_html.xsl` 
3. Use the output from 2 to generate the PDF with `rev5update-emulator_fo.xsl`

Note: previous testing issues with using Oxygen *from Linux* and the checks missing in the Table C in the Appendix were alleviated by [loading the Calibri font into my WSL2 Ubuntu Linux instance this way](https://gist.github.com/aj-stein-nist/9719ef0f4223d29fa690962e219f2c9e). Oxygen and Apache FOP integration does not report errors when fonts are requested but not in the operating system, even if the PDF generation specifically requires them. This is probably a convenience feature, but that is why the proper font was not embedded and checks became `#` only on my system. (I leave this as a note for other devs or future devs needing to review this work and have a head scratching moment.)

**Comment by wendellpiez** [comment]
Next steps for me:

- [x] Test all WIP XSLTs with some catalogs
- [x] Comments?
- [x] Consider renaming resources
- [x] Test everything again (post rename)
- [x] Commit changes to a stable `2022-template-rendition` branch in `usnistgov` repo for safekeeping
- [x] Remove the Rev 5 update customization layer and commit again to a 'march2023-update` branch
- [ ] Make a PR from this branch into `main`

?

File: publish/nist-emulation/rev5update_html.xsl [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
     xmlns:xs="http://www.w3.org/2001/XMLSchema"
     xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    xmlns:o="http://csrc.nist.gov/ns/oscal/1.0"
```
Comment (wendellpiez) [inline publish/nist-emulation/rev5update_html.xsl]: Oops, this duplicates line 8 below.

**Review by wendellpiez — COMMENTED** [review]
Alas it won't run the FO with the duplicate namespace declaration (attribute) in the FO XSLT.

File: publish/nist-emulation/rev5update_html.xsl (hunk 1 above)
Comment (aj-stein-nist) [reply publish/nist-emulation/rev5update_html.xsl]: .... oh!

Comment (aj-stein-nist) [reply publish/nist-emulation/rev5update_html.xsl]:
```suggestion
```

**Comment by aj-stein-nist** [comment]
> Alas it won't run the FO with the duplicate namespace declaration (attribute) in the FO XSLT.

Apologies, that should be corrected now. Thanks for catching that in review, my rebase skills are only so-so.

**Pull request merged**